### DISCUSSION TITLE
Lotes não encontrados  Lots not found

### DISCUSSION THREAD

**Discussion by ra339282** [Chia-Network/chia-blockchain#4358] [Support]
Lotes não encontrados tinha 23 lotes, fiz mais 3 e não aprecia, agora aparece os 3 e não identifica os 23???

Lots not found there were 23 lots, I made 3 more and do not appreciate it, now the 3 appear and do not identify the 23 ???

**Comment by lvcivs** [comment]
Are they in the same directory? 

Have you tried re-adding the directory through the GUI, in the plots tab?

**Reply by maikrodrigues** [reply]
Yes

**Comment by maikrodrigues** [comment]
Estou com o mesmo problema, já removi o caminho do diretório e adicionei novamente mas ainda persisti o em aparece lotes não encontrados.

**Comment by ra339282** [comment]
Amigos eu estava conectado na hpool apos eu ecluir a chave e criar uma nova e não fonesee mais a eles, os lotes não sincronizaram, acomteceu com voces tambem? ja fazerm 4 dias e nada ja revirei formas de restabeleser o sincronismo.

**Comment by cristianorandrade** [comment]
Estou com o mesmo problema. Aconteceu após exclusão da chave de 24 letras e criação de uma nova. Mesmo tendo recuperado a chave anterior não reconhece mais os lotes. Já exclui e inclui novamente o diretório, mas o programa não reconhece os lotes antigos.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
